examples/tutorials/starting/observation_clustering.py

```diff
@@ -0,0 +1,193 @@
+    obs_zenith.append(obs.get_pointing_altaz(time=obs.tmid).zen.deg)
+    obs_muoneff.append(obs.obs_info["MUONEFF"])
+
+print(f"{np.min(obs_zenith):.2f} deg < zenith angle < {np.max(obs_zenith):.2f} deg")
+print(f"{np.min(obs_muoneff):.2f} < muon efficiency < {np.max(obs_muoneff):.2f}")
+
+
+######################################################################
+# Manual grouping of observations
+# -------------------------------
+#
+# Here we can plot the zenith angle vs muon efficiency of the observations.
+# We decide to group the observations according to their zenith angle.
+# This is done manually as per a user defined cut, in this case we take the
+# median value of the zenith angles to define each observation group. These
+# are shown visually below.
+
+# This type of grouping can be utilised according to different parameters i.e.
+# zenith angle, muon efficiency, offset angle. The quantity chosen can therefore
+# be adjusted according to each specific science case.
+#
+
+fix, ax = plt.subplots(1, 1, figsize=(7, 5))
+obs_A = Observations([])
+obs_B = Observations([])
+for obs in observations:
+    zenith = obs.get_pointing_altaz(time=obs.tmid).zen.deg
+    if zenith < np.median(obs_zenith):
+        ax.plot(zenith, obs.obs_info["MUONEFF"], "d", color="red")
+        obs_A.append(obs)
+    if zenith > np.median(obs_zenith):
+        ax.plot(zenith, obs.obs_info["MUONEFF"], "o", color="blue")
+        obs_B.append(obs)
+ax.set_ylabel("Muon efficiency")
+ax.set_xlabel("Zenith angle (deg)")
+ax.axvline(np.median(obs_zenith), ls="--", color="black")
+
+
+######################################################################
+# This shows the observations grouped by zenith angle. The diamonds
+# are observations which have a zenith angle less than the median value,
+# whilst the circles are observations above the median.
+#
+# `obs_A` and `obs_B` are both `~gammapy.data.Observations` objects which
+# can be utilised in the usual way to show the various properties of the
+# observations i.e. see the :doc:`/tutorials/data/cta` tutorial.
+#
+
+
+######################################################################
+# Hierarchical clustering of observations
+# ---------------------------------------
+#
+# This method shows how to cluster observations based on their IRF quantities,
+# in this case those that have a similar edisp and psf. The
+# `gammapy.data.utils.get_irfs_features` is utilised to achieve this. The
+# observations are then clustered based on this criteria using
+# `gammapy.utils.cluster.hierarchical_clustering`. The idea here is to minimise
+# the variance of both edisp and psf within a specific group to limit the error
+# on the quanitity when they are stacked at the dataset level.
+#
+# In this example, the irf features are computed for the `edisp-res` and
+# `psf-radius` at 1 TeV. This is stored as a `astropy.table.table.Table`, as shown below.
+#
+
+source_position = SkyCoord.from_name("PKS 2155-304")
```

Comment:
please give the coordinate directly. I think we found previously that `SkyCoord.from_name` was taking a long time in the CI builds. Maybe this is what @morcuended was referring to ?